content/docs/2.10/operate/tls-certificates.md

```diff
@@ -0,0 +1,13 @@
++++
+title = "Use your own TLS Certificates"
```

Comment:
Let's call this "Security" with "Use your own TLS Certificates" as a section so we can add more topics in the future.

That way, this document is more of a catch-all for security topics and we don't have to break the URL in the future (assuming we also rename this file to `security.md`)

Reply:
I like the idea